### ISSUE TITLE
[bug]: The vertical align example's text in the documentation doesn't have enough contrast

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5249]
### 🐞 Describe the bug

In the [Utilities > Vertical align](https://next.design-system.post.ch/?path=/docs/cf01f6d1-970f-444e-aaa9-8a96c25cc8b2--docs) docs, the red `text` in the example does not have enough contrast.

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/b31df2dc-5af9-4f3d-b3f7-83e2591f2c4f)
